images/app/helpers/refinery/admin/images_helper.rb

```diff
@@ -19,6 +19,12 @@ def thumbnail_urls(image)
 
         thumbnail_urls
       end
+
+      # We show the title from the next available locale
+      # if there is no title for the current locale
```

Comment:
Agree, only when a fallback is configured should we fallback, and globalize should do that.